cmd/gopass-summon-provider/summon-provider.go

```diff
@@ -24,10 +25,24 @@ func (s *gc) Get(c *cli.Context) error {
 	ctx := ctxutil.WithGlobalFlags(c)
 	ctx = ctxutil.WithNoNetwork(ctx, true)
 	path := c.Args().Get(0)
+	key := c.Args().Get(1)
+	if key != "" {
+		ctx = action.WithKey(ctx, key)
+	}
 	secret, err := s.gp.Get(ctx, path, "latest")
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(Stdout, secret.Password())
+
+	if key == "" {
```

Comment:
This is not how I'd do this.

What about:

```
if key := c.Args().Get(1); key != "" {
  if val, found := secret.Get(key); found {
   fmt.Fprintln(Stdout, val)
   return
  }
  fmt.Fprintln(Stdout, "Error")
}
```

Reply:
That looks easier indeed, thanks.